src/d3a/models/market/__init__.py

```diff
@@ -124,6 +124,10 @@ def _create_fee_handler(self, grid_fee_type, grid_fees):
     def _is_constant_fees(self):
         return isinstance(self.fee_class, ConstantGridFees)
 
+    @property
+    def get_open_bids_and_offers(self):
```

Comment:
I think that, when defining properties' names, nouns are more suitable than verbs. So what would you think about renaming this to something like `open_bids_and_offers`?